docker/install_docker.sh

```diff
@@ -1,25 +0,0 @@
-#!/bin/bash
```

Comment:
This script is needed, as anyone using the pre-built containers needs to be able to launch Triton from within the Morpheus container (#552 ). 

The existing docker build doesn't include docker itself, and the documentation currently instructs users on running this script.

Reply:
I see this was moved to the utilities repo, we just need to ensure that the  `docs/source/developer_guide/contributing.md` and `docs/source/getting_started.md` docs that reference this script are updated

Reply:
Updated.